components/custom-network/new-network-stepper.tsx

```diff
@@ -57,11 +64,29 @@ function NewNetwork() {
     { id: 1, name: t("custom-network:modals.loader.steps.changing-dispute-percentage") },
     { id: 1, name: t("custom-network:modals.loader.steps.changing-council-amount") },
     { id: 2, name: t("custom-network:modals.loader.steps.add-to-registry") },
-    { id: 3, name: t("custom-network:modals.loader.steps.sync-web-network") }
+    { id: 3, name: t("custom-network:modals.loader.steps.sync-web-network") },
+    { id: 3, name: t("custom-network:modals.loader.steps.sync-chain-id") }
   ];
 
   async function handleCreateNetwork() {
     if (!state.currentUser?.login || !state.currentUser?.walletAddress || !state.Service?.active) return;
+
+    const signedMessage = await signMessage(WANT_TO_CREATE_NETWORK)
+      .catch(error => {
```

Comment:
add this catch to the `signMessage` logic, so we don't need to repeat it elsewhere :)